packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -472,6 +472,11 @@ class _TooltipState extends State<Tooltip> with SingleTickerProviderStateMixin {
 
   @override
   Widget build(BuildContext context) {
+    // If message is empty then no need to create a tooltip overlay to show
+    // the empty black container so just return the wrapped child as is
+    if (widget.message.isEmpty) {
+      return widget.child!;
```

Comment:
I think that it's pretty often that the tooltip wouldn't have a child.  Perhaps you could do something like this instead, so that it wouldn't crash if there were no child.

```
return widget.child ?? const SizedBox();
```

Reply:
You are absolutely right. I was a little innattentive and didn't noticed that `child` is not required parameter.